src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -28,21 +29,24 @@ class JsonAdaptedPerson {
     private final String phone;
     private final String email;
     private final String address;
-    private final List<JsonAdaptedTag> tags = new ArrayList<>();
+    private final String remark;
+    private final List<JsonAdaptedTag> tagged = new ArrayList<>();
```

Comment:
Personally I would retain `tags` instead of renaming the variable to `tagged` since we are referring to a list of tags. `tagged` doesn't quite imply list of tags - I would infer it to be a `boolean` variable (although `isTag` better implies `boolean` i know)

Reply:
Hmm, I agree with your reasoning. I changed it because I thought the past sense made more sense, but I shall revert this change, thanks!